vip-workflow.php

```diff
@@ -16,28 +16,34 @@
 
 namespace VIPWorkflow;
 
-if ( ! defined( 'VIP_WORKFLOW_LOADED' ) ) {
-	define( 'VIP_WORKFLOW_LOADED', true );
-
-	// ToDo: Add a check for the WP version as well.
-	// ToDo: When 6.4 is our min version, switch to wp_admin_notice.
-	if ( version_compare( phpversion(), '8.0', '<' ) ) {
-		add_action( 'admin_notices', function () {
-			?>
-			<div class="notice notice-error">
-					<p><?php esc_html_e( 'VIP Workflow requires PHP 8.0+.', 'vip-workflow' ); ?></p>
-				</div>
-			<?php
-		}, 10, 0 );
-		return;
-	}
+if ( defined( 'VIP_WORKFLOW_LOADED' ) ) {
+	return;
+}
 
-	// Define contants
-	define( 'VIP_WORKFLOW_VERSION', '0.2.0' );
-	define( 'VIP_WORKFLOW_ROOT', __DIR__ );
-	define( 'VIP_WORKFLOW_URL', plugins_url( '/', __FILE__ ) );
-	define( 'VIP_WORKFLOW_SETTINGS_PAGE', add_query_arg( 'page', 'vw-settings', get_admin_url( null, 'admin.php' ) ) );
-	define( 'VIP_WORKFLOW_REST_NAMESPACE', 'vip-workflow/v1' );
+define( 'VIP_WORKFLOW_LOADED', true );
 
-	require_once VIP_WORKFLOW_ROOT . '/class-workflow.php';
+// ToDo: Add a check for the WP version as well.
+// ToDo: When 6.4 is our min version, switch to wp_admin_notice.
+if ( version_compare( phpversion(), '8.0', '<' ) ) {
+	add_action( 'admin_notices', function () {
+		?>
+		<div class="notice notice-error">
+				<p><?php esc_html_e( 'VIP Workflow requires PHP 8.0+.', 'vip-workflow' ); ?></p>
+			</div>
+		<?php
+	}, 10, 0 );
+	return;
 }
+
+// Define contants
+define( 'VIP_WORKFLOW_VERSION', '0.2.0' );
+define( 'VIP_WORKFLOW_ROOT', __DIR__ );
+define( 'VIP_WORKFLOW_URL', plugins_url( '/', __FILE__ ) );
+define( 'VIP_WORKFLOW_SETTINGS_PAGE', add_query_arg( 'page', 'vw-settings', get_admin_url( null, 'admin.php' ) ) );
+define( 'VIP_WORKFLOW_REST_NAMESPACE', 'vip-workflow/v1' );
+
+// Main plugin class
+require_once VIP_WORKFLOW_ROOT . '/class-workflow.php';
+
+// Modules
+require_once VIP_WORKFLOW_ROOT . '/modules/preview/preview.php';
```

Comment:
I've moved all of the base requires to this file, rather than `class-workflow.php`. We really just need a centralized place to include these top-level imports, and this is a simpler and much less busy file than `class-workflow.php`.

Classes that are particular to a module (e.g. `PreviewEndpoint`) are required by the module that uses them.